cleanlab_studio/cli/dataset/schema_types.py

```diff
@@ -13,6 +13,7 @@ class DataType(Enum):
     integer = "integer"
     float = "float"
     boolean = "boolean"
+    media = "media"
```

Comment:
why do we need a media type here? these should always be strings -- the interpretation of these strings as media is encapsulated by the `FeatureType`

Reply:
Yeah this is true. It's definitely not necessary to have a media DataType here. Mostly added it so it'd be easy to know which FeatureTypes should be interpreted as media, but there are other ways we could achieve that

Reply:
Yeah, totally get that. I'd definitely prefer a different solution here -- it seems nice to keep the meaning of `DataType` pure